src/connectors/current-refinements/__tests__/connectCurrentRefinements-test.ts

```diff
@@ -394,118 +439,112 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/current-ref
   });
 
   describe('Rendering options', () => {
-    let helper;
+    let helper: AlgoliaSearchHelper;
 
     beforeEach(() => {
-      helper = jsHelper({}, '', {
+      helper = algoliasearchHelper(createSearchClient(), '', {
         facets: ['facet1', 'facet2', 'facet3'],
       });
-      helper.search = () => {};
+      helper.search = jest.fn();
     });
 
-    it('provides a `refine` function', () => {
```

Comment:
this is because you first removed the root refine function right?

Reply:
I tested it here: https://github.com/algolia/instantsearch.js/blob/db681a241507c5f9caab63fb45c63f605e817080/src/connectors/current-refinements/__tests__/connectCurrentRefinements-test.ts#L81-L82